content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -480,8 +480,8 @@ There are some implicit conventions worth noting here:
 - The scheduler bypasses any nodes that don't have any `topologySpreadConstraints[*].topologyKey`
   present. This implies that:
 
-  1. any Pods located on those bypassed nodes do not impact `maxSkew` calculation - in the
-     above example, suppose the node `node1` does not have a label "zone", then the 2 Pods will
+  1. any Pods located on those bypassed nodes do not impact `maxSkew` calculation - [in the
+     above example](#example-conflicting-topologyspreadconstraints), suppose the node `node1` does not have a label "zone", then the 2 Pods will
```

Comment:
```suggestion
  1. any Pods located on those bypassed nodes do not impact `maxSkew` calculation - in the
     above [example](#example-conflicting-topologyspreadconstraints), suppose the node `node1` does not have a label "zone", then the 2 Pods will
```
even better. The link is to an example, not to an in the above example.

Reply:
@sftim Thanks for the suggestion